internal/server/data/migrations.go

```diff
@@ -614,3 +614,44 @@ CREATE UNIQUE INDEX IF NOT EXISTS idx_identities_verified ON identities (organiz
 		},
 	}
 }
+
+func cleanCrossOrgGroupMemberships() *migrator.Migration {
```

Comment:
Might be worth checking prod first to see if this migration finds any groups, so that we know if anyone was impacted or not.